### DISCUSSION TITLE
[Script request] Tarpit LXC

### DISCUSSION THREAD

**Discussion by garcipat** [community-scripts/ProxmoxVE#302] [Request script] [answered]
### Application Name

Tarpit

### Website

https://github.com/Snawoot/ssh-tarpit

### Description

Tarpit is a honeypot like SSH target that is intended to keep attackers busy as long as possible to make the internet a little bit more secure and wast their time.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
Rejected: Security concerns due to the old (> 3 years not updated) code base.
